proc_scripts/CPMG_phasing.py

```diff
@@ -18,13 +20,22 @@ def center_CPMG_echo(s, axis='t2',fl=None):
     """
     echo_center = hermitian_function_test(s, fl=fl)
     logger.info(strm("echo center is",echo_center))
+    return echo_center 
+def center_echo(s, echo_center, axis='t2',fl=None):
```

Comment:
this guy needs documentation.  I'm not going to read the code until I know what it's supposed to do.

Reply:
this has a very different purpose -- the hermitian function test is the *find* the center of the echo.  This routine is essentially an alternative to and FID slicing -- here we are trying to slice a symmetric echo (same amount of negative time as positive time).  Resolve this once you read it.

Reply:
I edited that in 8f2de892

Reply:
edited to be more accurate